Development.md

```diff
@@ -0,0 +1,235 @@
+
+- Feature branches for epics will be created on the production repo (`xai-foundation/sentry`) (this requires write access on that repo).
+- Working branches will be created **from** `origin/xai-foundation/sentry/feature-branch` **in** `xai-foundation/sentry-development:working-branch`
+- PRs get created on `xai-foundation/sentry-development:working-branch` back into `xai-foundation/sentry/feature-branch`
+- Every PR links to its story and has a description on
+  - What has been done
+  - How has it been tested
+- Every story links to it's PR for it to be marked as finished
+- Once the PR is approved the PR requester will merge and deploy, mark the story as delivered and add info for testing to the story
+
+## Local development
+
+Currently the monorepo can be installed and built using **pnpm** except `web-staking` that has yet to be integrated and needs to be managed with **npm** (https://www.pivotaltracker.com/story/show/187887214)
+
+[NX](https://nx.dev/features/run-tasks) can be used for local script management, currently for every app except `web-staking`
+
+Run the different apps by their commands in the root's `package.json` like
+
+- `pnpm cli` for running the CLI
+- `pnpm web` for running sentry-xai.games locally
+- `pnpm desktop` for running the desktop app locally
+
+For local release building and signing SSL.com access is requires. Currently the credentials are set in the github repositories sentry & sentry-development, however, sentry.development is currently breaking on sign.
+
+A local release and a github release can be created without signing the release.
+Currently only the windows release gets signed.
+
+---------------------------------
+
+# Deployments & Releases
+
+## Develop Environment
+
+### Operator & Web-Connect
+
+Push a **tag** on `xai-foundation/sentry-development:working-branch` to trigger the github actions. This will deploy
+
+- **web-connect** to https://xai-foundation.github.io/sentry-development/ (github pages)
+- **CLI & Desktop App (all OS)** to https://github.com/xai-foundation/sentry-development/releases
+
+THIS WILL TRIGGER THE AUTO UPDATE FOR ALL PREVIOUS DESKTOP CLIENTS !
+
+**Important sepolia testnet updates:**
+- `packages\core\src\config.ts` needs to be updated, current sepolia testnet config can be found here https://github.com/xai-foundation/sentry-development/blob/build-sepolia/packages/core/src/config.ts
+- `useBlockIp()` hook in `apps/web-connect` needs to set `setBlocked(false);` for any user to disable geo-block
+- currently we still need to comment out hardcoded checks for chainIds and RPCs in web-connect and core. For this we can use the prepared branch `xai-foundation/sentry-development:build-sepolia`. For local testing and test deploys new branches can be created off `build-sepolia` and tags pushed from there.
+- remove `compareWithCDN` in  `packages\core\src\operator\operatorRuntime.ts`, there is no sepolia public node that publishes to the CDN
```

Comment:
Should we note this change is only required if you're working with/on the operator?

For example, if I'm only working on web-connect, I don't really need to update the compareWithCDN?

Same for the others? Maybe each app that needs config update(s) get's a header with bullet points under?

Reply:
I would not differentiate as you will deploy a release build and it will include the operator and desktop app. If we don't always update that on the sepolia release we would have to track specific versions where only web-connect is working for sepolia but the operator is not or is behaving weird.